src/btree/bt_discard.c

```diff
@@ -531,3 +531,17 @@ __wt_free_update_list(WT_SESSION_IMPL *session, WT_UPDATE **updp)
     }
     *updp = NULL;
 }
+
+/*
+ * __wt_free_update_next_list --
+ *     Free the WT_UPDATE next linked list.
```

Comment:
Please add better naming for `upd` parameter and a description of the expectations on the passed in update, and why the subsequent one is the one that is actually freed.

Reply:
Fixed.